uint16 statusword
uint8 error_register
